packages/dnb-design-system-portal/src/docs/uilib/components/avatar/Examples.tsx

```diff
@@ -0,0 +1,282 @@
+/**
+ * UI lib Component Example
+ *
+ */
+
+import ComponentBox from 'dnb-design-system-portal/src/shared/tags/ComponentBox'
+import {
+  card as Card,
+  account_card as AccountCard,
+  confetti as Confetti,
+} from '@dnb/eufemia/src/icons'
+import { Avatar, Icon, Logo } from '@dnb/eufemia/src'
+
+export const AvatarSizeDefault = () => (
+  <ComponentBox hideCode data-visual-test="avatar-size-default">
+    <Avatar.Group label="Persons:">
+      <Avatar>Ola Nordmann</Avatar>
+    </Avatar.Group>
+  </ComponentBox>
+)
+
+export const AvatarSizeSmall = () => (
+  <ComponentBox hideCode data-visual-test="avatar-size-small">
+    Text{' '}
```

Comment:
```suggestion
    Text
```

I'll look into if we really need this `{' '}`. I see we have a few `{' '}` in different Examples.tsx files, but I'm not sure if we actually need them, or if they were intentional, I feel that sometimes they randomly pop up prettier and different "formatting tools".

Reply:
Yeah, we need a space there. So there are not too many other options actually.

Reply:
Okok, if we need a space, then it's of course correct 👍 
I just assumed the space was not needed in `AvatarSizeSmall`, as we don't have it in `AvatarSizeMedium`, `AvatarSizeLarge`, and `AvatarSizeXLarge`.

Reply:
Never mind, it's all good.
I think this is good yes ✅